src/lib/swapper/swappers/ThorchainSwapper/utils/getQuote/getQuote.ts

```diff
@@ -101,3 +104,23 @@ export const getQuote = async ({
     return Ok(data)
   }
 }
+
+export const getQuote = async (
+  input: GetQuoteArgs,
+): Promise<Result<ThornodeQuoteResponseSuccess, SwapErrorRight>> => {
+  const initialQuoteResult = await _getQuote(input)
+
+  if (initialQuoteResult.isErr()) return initialQuoteResult
+
+  const initialQuote = initialQuoteResult.unwrap()
+
+  // refetch quote without affiliate fee if it's less than the thorchain outbound fee
+  if (
+    input.affiliateBps !== undefined &&
+    initialQuote.fees.affiliate < THORCHAIN_OUTBOUND_FEE_RUNE
+  ) {
```

Comment:
This logic doesn't work, as `affiliate` here is the amount of the outbound (receive) asset in base units, and so can't be compared to the RUNE fee amount.

Fortunately I think we can overcome this pretty easily - the `fees` object contains `outbound`, which as far as I can tell, is the outbound fee amount denominated in the outbound asset.

So we can simply do `initialQuote.fees.affiliate <= initialQuote.fees.outbound`.

Reply:
Ah ok. How sure are you on this one? The documentation says all affiliate fees are paid in rune, but it makes sense that they specify it in the quote denominated in the outbound asset.

I don't think it's in base units though, it's probably Thor precision (1e8?)

Reply:
Thinking on this more I'm pretty certain you're right. Will fix

Reply:
Oh, it's thor base units, yes.